R/render_planning_doc.R

```diff
@@ -0,0 +1,17 @@
+#' Render planning doc
```

Comment:
This is confusing together with `render_plan_doc.R`. Looks like something got duplicated?

Reply:
deleted `render_plan_doc.R`, thanks for noticing